sendgrid/helpers/mail/mail.py

```diff
@@ -251,6 +251,36 @@ def add_custom_arg(self, custom_arg):
             self._custom_args = []
         self._custom_args.append(custom_arg)
 
+################################################################
+
+
+class SendGridMessage(Mail):
+    def __init__(self, 
+                from_email=None,
+                subject=None, 
+                to_email=None, 
+                plain_text_content=None, 
+                html_content=None):
+        self.from_email = from_email
+        self.subject = subject
+        self.to_email = to_email
+        self.plain_text_content = plain_text_content
+        self.html_content = html_content
+
+    def get(self):
+        if self.plain_text_content is None:
+            self.plain_text_content = ""
+        if self.html_content is not None:
+            self.content = Content("text/html", self.plain_text_content + self.html_content)
+        else:
+            self.content = Content("text/plain", self.plain_text_content)
```

Comment:
The plain text only gets rendered if there is no HTML content.

When we build the final request object, we just need to make sure that the "text/plain" content always comes first. See [here](https://github.com/sendgrid/sendgrid-csharp/blob/master/src/SendGrid/Helpers/Mail/SendGridMessage.cs#L1464) for an example.

Thanks!